tap_google_ads/streams.py

```diff
@@ -1022,16 +1022,16 @@ def initialize_reports(resource_schema):
                 "campaign_criterion_criterion_id",
             },
         ),
-        "click_performance_report": ReportStream(
-            report_definitions.CLICK_PERFORMANCE_REPORT_FIELDS,
-            ["click_view"],
-            resource_schema,
-            ["_sdc_record_hash"],
-            {
-                "clicks",
-                "click_view_gclid",
-            },
-        ),
+        # "click_performance_report": ReportStream(
+        #     report_definitions.CLICK_PERFORMANCE_REPORT_FIELDS,
+        #     ["click_view"],
+        #     resource_schema,
+        #     ["_sdc_record_hash"],
```

Comment:
Are you sure we don't need this?

Reply:
Working on the solution to include this.